src/colab/Initialize_SKAI_Colab_Kernel.ipynb

```diff
@@ -0,0 +1,249 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Initialize SKAI Colab Kernel",
+      "provenance": [],
+      "private_outputs": true,
+      "collapsed_sections": [
+        "KzSkXvGX6IAz"
+      ],
+      "history_visible": true,
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/google-research/skai/blob/skai-colab-0000002/src/colab/Initialize_SKAI_Colab_Kernel.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Initialize SKAI Colab Kernel\n",
+        "This notebook contains a set of commands to run on a new custom colab kernel VM. These commands will pre-install all the necessary dependencies in order to run the SKAI pilot notebook.\n",
+        "\n",
+        "To use:\n",
+        "1. **Open notebook in Playground mode. This is so that you don't accidentally save settings changes you make.**\n",
```

Comment:
If the user opens the notebook from the github repo, any changes they make cannot be saved anyway. In fact the "Open in Playground Mode" menu item doesn't even exist when the notebook is opened in this way. So we should just delete this line.

Reply:
I agree. Maybe we should nevertheless mentioned that the user has to saved a copy and his/her changes in his/her own Google drive or GitHub repo. What do you think ?

Reply:
I am assuming that users will just need to open the notebook, change 3 parameters, run it, and be done with it. I don't see the need for the user to save a copy of the notebook.

Reply:
The last commit completes the changes and include also a change if SKAI_COMMIT is left empty